src/Orleans/Runtime/GrainReference.cs

```diff
@@ -378,7 +398,7 @@ private void CallClientInvokeCallback(InvokeMethodRequest request)
         }
 
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
-        private static void ResponseCallback(Message message, TaskCompletionSource<object> context)
+        private void ResponseCallback(Message message, TaskCompletionSource<object> context)
```

Comment:
Ah this is one of those things which changes in a subsequent PR (non-static SerializationManager). In this case, it's `message.BodyObject` which will change. My bad. I can revert those changes and push them into the later PR, but I wanted to extract as much as possible from that PR since it's too heavy for people to review.
![image](https://cloud.githubusercontent.com/assets/203839/23045163/b4e744b8-f4f7-11e6-8551-69977f06f476.png)